roles/kubernetes-apps/external_provisioner/local_volume_provisioner/tasks/main.yml

```diff
@@ -1,8 +1,10 @@
 ---
+- set_fact:
+    local_volume_provisioner_storage_classes: "{{ local_volume_provisioner_storage_class_maps.keys() }}"
```

Comment:
Doesn't this fact name conflict with the name list of maps variable in the documentation?

Reply:
Good catch! I seem to have rebased a little to enthusiastic, please hold off merging. I will run some more tests tomorrow 🤓

Reply:
Good catch! I seem to have rebased a little to enthusiastic, please hold off merging. I will run some more tests tomorrow 🤓